docker/Dockerfile.contrib

```diff
@@ -1,12 +1,24 @@
-FROM node:erbium-stretch AS build
-USER root
-RUN apt-get update && apt-get install -y jq
-RUN mkdir -p /usr/local/lib/node_modules && chown node /usr/local/lib/node_modules
-RUN yarn global add yalc
+ARG SRC=git-clone-src
+
+##### GIT CLONE #####
+FROM node:erbium-buster AS git-clone-src
+ARG ZWJ_BRANCH=master
+ARG Z2M_BRANCH=master
+USER node
+WORKDIR /home/node
+RUN mkdir -p node-zwave-js zwavejs2mqtt
+RUN curl -sL https://github.com/zwave-js/node-zwave-js/tarball/${ZWJ_BRANCH} |  tar -xz --strip 1 -C node-zwave-js
+RUN curl -sL https://github.com/zwave-js/zwavejs2mqtt/tarball/${Z2M_BRANCH} |  tar -xz --strip 1 -C zwavejs2mqtt
 
+##### LOCAL SOURCE #####
+FROM node:erbium-buster AS local-copy-src
```

Comment:
Would this step fail if you don't have a local copy? Also can this step be skipped if the SRC is git?

Reply:
This will fail if you don't have a local copy, but the default is github src. It will be skipped if you use buildkit, but will fail if you don't use buildkit and don't have a local copy. I think that's okay.

Reply:
i.e. it only fails if you actively specify SRC to be local copy, and that you haven't got a local copy.